### DISCUSSION TITLE
Font options for the documentation

### DISCUSSION THREAD

**Discussion by pavel-voronin** [marcj/deepkit#320] [Polls]
It's hard for me to read documentation as it uses serif typeface. The most of popular resources use handy fonts. E.g.: github, node.js, typescript, mdn, react, vue, express.js and so on.

What do you think, should we consider to change the font to one that is dyslexia friendly or provide an option to choose between few fonts?

Or maybe, leave it and let people with certain disorders solve their problem on their own (e.g. using extensions for mobile browsers if it possible)?

This is not a bad idea to align documentation style with common practice. What do you think?

**Comment by marcj** [comment]
Font is changed to a sans serif one. Better for you now?

**Reply by pavel-voronin** [reply]
It's great! Thank you

**Reply by nikolay** [reply]
Thank you, it's better, but a little hard to read for me personally.

**Reply by marcj** [reply]
@nikolay what is the reason?

**Reply by jackgdll** [reply]
@marcj I find it a little hard to read mostly because there are a lot big paragraphs of text and quite a bit of repeated information. I think a lot of passages could be shortened or better formatted.

I understand this is not a small task and would love to contribute but given that I only discovered this library last week (and loving it so far!) I'll take time to familiarise myself with Deepkit first

**Reply by marcj** [reply]
maybe we can automatically optimise it using GPT4, multiple paragraphs at once, but haven't tried it yet
